project.clj

```diff
@@ -48,7 +48,7 @@
                                   [ring-cors "0.1.11"]
                                   [compojure "1.6.0"]]
 
-                   :plugins [[lein-ring "0.12.0"]]
+                 $ :plugins [[lein-ring "0.12.5"]]
```

Comment:
Is this `$` supposed to be here?

Reply:
Sorry for the delay either, I never received any notification.

 Definitely not !